### PR TITLE
#1 Start

### PR DESCRIPTION
**Pull request by KrasnyiM**

### CODE REVIEW

File: OrionWebAPI.sln
Comment (ryzhkovAndruha) [inline OrionWebAPI.sln:1]: Shouldn't this file be under gitignore?

File: DbGame/User.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
```
Comment (ryzhkovAndruha) [inline DbGame/User.cs:6]: Remove unnecessary usings

File: DbGame/GameDto.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (ryzhkovAndruha) [inline DbGame/GameDto.cs:5]: Remove unnecessary usings

File: DbGame/UserRequest.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (ryzhkovAndruha) [inline DbGame/UserRequest.cs:5]: Remove unnecessary usings

File: OrionWebAPI/AutoMapperProfile.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
```
Comment (ryzhkovAndruha) [inline OrionWebAPI/AutoMapperProfile.cs:6]: Remove unnecessary usings

File: DbGame/Accounting.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (ryzhkovAndruha) [inline DbGame/Accounting.cs:5]: Remove unnecessary usings

File: DbGame/ApplicationContext.cs [hunk 6]
```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
```
Comment (ryzhkovAndruha) [inline DbGame/ApplicationContext.cs:8]: Remove unnecessary usings

File: DbGame/ApplicationContext.cs [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+            //        j => j
+            //        .HasOne(pt => pt.Game)
+            //        .WithMany(p => p.Accounting)
+            //        .HasForeignKey(pt => pt.GameId),
+            //        j =>
+            //        {
+            //            j.Property(pt => pt.DateTime).HasDefaultValue(DateTime.Now);
+            //            j.HasKey(t => new { t.UserId, t.GameId });
+            //            j.ToTable("Accountings");
+            //        });
```
Comment (ryzhkovAndruha) [inline DbGame/ApplicationContext.cs:44]: Good practice is when you don't have commented code. Either delete it or uncomment it

File: DbGame/DbGame.csproj
Comment (ryzhkovAndruha) [inline DbGame/DbGame.csproj:1]: Shouldn't this file be under gitignore?

File: OrionWebAPI/OrionWebAPI.csproj
Comment (ryzhkovAndruha) [inline OrionWebAPI/OrionWebAPI.csproj:1]: Shouldn't this file be under gitignore?

File: OrionWebAPI/Program.cs [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+// Add services to the container.
+builder.Services.AddDbContext<ApplicationContext>(options => {
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"));
+    });
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+//builder.Services.AddScoped<ApplicationContext>();
```
Comment (ryzhkovAndruha) [inline OrionWebAPI/Program.cs:16]: Why this code is commented?

File: OrionWebAPI/appsettings.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
+{
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost;Database=Orion;Trusted_Connection=True;TrustServerCertificate=True;"
+  },
```
Comment (ryzhkovAndruha) [inline OrionWebAPI/appsettings.json:4]: This is very sensetive information, that should not be pushed to your github. Read about user secrets

**Review by ryzhkovAndruha — CHANGES_REQUESTED** [review]
Read the comments that should be changed

File: OrionWebAPI/Program.cs (hunk 8 above)
Comment (KrasnyiM) [reply OrionWebAPI/Program.cs:16]: I forgot to delete this code line. I practice different ways to inject objects, and then I understand to work without explicit implementation. If I understand all right.

**Review by ryzhkovAndruha — APPROVED** [review]

**Pull request merged**